### PR TITLE
fix: deduplicating nested install structure

### PR DESCRIPTION
**Pull request by ethangj**
This removes the unnecessary second tier of install instructions and creates a single install parent.

### CODE REVIEW

**Comment by ethangj** [comment]
@hferentschik Struggling to figure out how to get this one to pass

**Comment by hferentschik** [comment]
> Struggling to figure out how to get this one to pass

There are still a few broken internal links which need to be addressed:

```
- ./tmp-website/docs/reference/components/docker-registry/index.html
  *  internally linking to docs/install-setup/boot/secrets/, which does not exist (line 0)
     <a href="docs/install-setup/boot/secrets/">Vault</a>
- ./tmp-website/docs/reference/components/vault/index.html
  *  internally linking to /docs/install-setup/boot/secrets/clouds/amazon/#configuring-dns-and-tls-on-eks, which does not exist (line 0)
     <a href="/docs/install-setup/boot/secrets/clouds/amazon/#configuring-dns-and-tls-on-eks">Configuring DNS and TLS on EKS</a>
  *  internally linking to /docs/install-setup/boot/secrets/clouds/google, which does not exist (line 0)
     <a href="/docs/install-setup/boot/secrets/clouds/google">Configuring DNS and TLS on GKE</a>
  *  linking to internal hash #configuring-dns-and-tls-on-eks that does not exist (line 0)
     <a href="/docs/install-setup/boot/secrets/clouds/amazon/#configuring-dns-and-tls-on-eks">Configuring DNS and TLS on EKS</a>
htmlproofer 3.15.3 | Error:  HTML-Proofer found 4 failures!
```

Once this is addressed the pull request should build and deploy. 

@ethangj, does this help? Or do you have issues locating and/or updating these links?

**Comment by ethangj** [comment]
@hferentschik are those not auto-generated? i can't actually find the referenced links on that page.

**Comment by hferentschik** [comment]
> are those not auto-generated? 

no, why do you think so? The command reference is generated, not the links mentioned above.

> i can't actually find the referenced links on that page.

Let me check ...

File: content/en/docs/install-setup/boot/newsletter/_index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: Jenkins X Community Newsletter
+linktitle: Newsletter
+type: docs
+weight: 50
+---
+
+<img src="https://gallery.mailchimp.com/d0c128ac1f69ba2bb20742976/images/279a413a-a5ac-4762-83ef-36e82133a9b9.png" class="img-fluid mx-auto d-block"/>
```
Comment (hferentschik) [inline content/en/docs/install-setup/boot/newsletter/_index.md:8]: How did that get in here? That must be a mistake

**Comment by hferentschik** [comment]
@ethangj, I found the culprits. Let's see how it goes. I also removed the newsletter directory. Not sure how that got in there!?

FYI, I suggest you fork the repo and create your pull request from your fork. We use/recommend the "fork and pull request" model when working with any repo. See also https://jenkins-x.io/community/code/#fork-the-repository or https://gist.github.com/Chaser324/ce0505fbed06b947d962. There are several reasons why this makes sense, but most importantly it decreases the risk that you accidentally push to master.

**Comment by ankitm123** [comment]
> FYI, I suggest you fork the repo and create your pull request from your fork. We use/recommend the "fork and pull request" model when working with any repo. See also https://jenkins-x.io/community/code/#fork-the-repository or https://gist.github.com/Chaser324/ce0505fbed06b947d962. There are several reasons why this makes sense, but most importantly it decreases the risk that you accidentally push to master.

:+1:  Should be added to the documentation contribution page (a link to https://jenkins-x.io/community/code/#fork-the-repository).

**Pull request merged**